src/iput.cpp

```diff
@@ -163,7 +163,7 @@ usage( FILE* _fout ) {
         "This option is only effective if the source is a directory and the -f ",
         "option is not used.",
         " ",
-        "The -Q option specifies the use of the RBUDP transfer mechanism which uses",
+        "The -Q option [Deprecated] specifies the use of the RBUDP transfer mechanism which uses",
```

Comment:
Same here - perhaps make the deprecation note a separate sentence?

Reply:
Done